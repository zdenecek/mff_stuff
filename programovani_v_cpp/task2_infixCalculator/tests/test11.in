5
(1)
(1)+(1)+(1)+(1)
((((((((6))))))))+(((5)))
((1)+(-4)-(5*5))
(5.6)