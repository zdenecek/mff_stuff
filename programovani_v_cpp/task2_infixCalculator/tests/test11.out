1
4
11
-28
5.6
